src/elements/loading-indicator/readme.md

```diff
@@ -1,32 +1,13 @@
 The `sbb-loading-indicator` is a component which can be used to indicate progress status
 or an ongoing activity which require some time to complete.
-
-### Variants
-
-The component has two different variants.
-
-In `window` mode, the component completely covers the parent element, preventing interaction with it.
-
-```html
-<sbb-loading-indicator variant="window"></sbb-loading-indicator>
-```
-
-While the `circle` mode can be used inline within another component (e.g. button);
-in this case the component adjusts its size to the parent font size.
-
-```html
-<sbb-button>
-  <sbb-loading-indicator slot="icon" variant="circle"></sbb-loading-indicator>
-  Click me
-</sbb-button>
-```
+The component completely covers the parent element, preventing interaction with it.
```

Comment:
Not sure, Davide suggested adding this, and I just thought I might have missed this detail

Reply:
Because the loading indicator doesn't have to cover any parent element. it's just an element which can be used like the consumer wants. @DavideMininni-Fincons What was your thought? Or am I misunderstanding it?

Reply:
I just took it from the old readme (see some lines below in the old file), I thought that was accidentally removed while migrating. 
If not, just remove it (sorry for the misunderstanding) :)

Reply:
I'd say this was wrong before already :)